packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.tsx

```diff
@@ -82,6 +82,7 @@ export class GroupHeader extends BaseComponent<IGroupDividerProps, IGroupHeaderS
             <button
               type='button'
               className={ css('ms-GroupHeader-check', styles.check) }
+              aria-label='group checkbox button'
```

Comment:
Please make this a property of `GroupHeader` rather than a hardcoded string.

Reply:
Accepted and updated.